components/ItemCripto/index.tsx

```diff
@@ -8,31 +8,56 @@ import {
   CriptoName,
   CriptoPercentage,
   ImageArrow,
+  ViewArrow,
+  CriptoPrice,
+  CriptoImg,
 } from './styles';
 
-const ItemCripto = ({item}: ItemCriptoProps) => (
-  <CriptoContainer>
-    <SeconContainer>
-      <Image source={item.img} />
-      <View>
-        <CriptoName>{item.name}</CriptoName>
-        <Text>{item.symbol}</Text>
-      </View>
-    </SeconContainer>
-    <View>
-      <CriptoName>{item.price}</CriptoName>
-      <CriptoPercentage color={item.percentage > 0}>
-        <ImageArrow
-          source={
-            item.percentage > 0
-              ? require('../../assets/img/greenarrow.png')
-              : require('../../assets/img/redarrow.png')
-          }
-        />
-        {Math.abs(item.percentage) + '%'}
-      </CriptoPercentage>
-    </View>
-  </CriptoContainer>
-);
+export const truncPrice = (n: any) => {
+  if (n === null) {
+    return 0;
+  }
+  if (n < 0) {
+    let t = n.toString();
+    let q = t.substring(1);
+    let regex = /(\d*.\d{0,2})/;
+    return q.match(regex)[0];
+  } else {
+    let t = n.toString();
+    let regex = /(\d*.\d{0,2})/;
+    return t.match(regex)[0];
+  }
+};
+
+const ItemCripto = ({item}: ItemCriptoProps) => {
+  return (
+    <Pressable>
+      <CriptoContainer>
+        <SeconContainer>
+          <CriptoImg source={item.img} />
+          <View>
+            <CriptoName>{item.name}</CriptoName>
+            <Text>{item.symbol}</Text>
+          </View>
+        </SeconContainer>
+        <View>
+          <CriptoPrice>{'$' + truncPrice(item.price)}</CriptoPrice>
```

Comment:
use item.price.toFixed(2) and delete truncPrice function

Reply:
Wow, i wish i new this function before haha. I will change that!